src/index.ts

```diff
@@ -20,10 +21,14 @@ export default async function createAuth0Client(options: Auth0ClientOptions) {
       'For security reasons, `window.crypto` is required to run `auth0-spa-js`.'
     );
   }
+  // safari 10 compat
+  if (!window.crypto.subtle && (<any>window.crypto).webkitSubtle) {
+    (<any>window.crypto).subtle = (<any>window.crypto).webkitSubtle;
```

Comment:
No non-Auth0 code can be running in the Window while we're doing this right?

Trying to decide if replacing a global is safe in this context or whether we should be creating our own variable to use instead to avoid non-Auth0 code getting confused.